lib/aptly/snapshot.rb

```diff
@@ -50,6 +50,23 @@ def publish(prefix, **kwords)
       Aptly.publish([{ Name: self.Name }], prefix, 'snapshot', kwords)
     end
 
+    # Lists all PublishedRepositories self is published in. Namely self must
```

Comment:
Seeing as this method is a verbatim copy of the Repository version I am wondering if we should have a module that gets included to provide this functionality instead of copying the method around?
```
module Aptly::Publishable
def published_in
#...
end
end

class Snapshot
include Publishable
end
```